packages/components/src/components/Alert/Alert.tsx

```diff
@@ -127,25 +134,25 @@ export const Alert: FC<AlertProps> = ({
   const contentBox: ViewStyle = {
     backgroundColor: backgroundColor,
     borderLeftColor: borderColor,
-    borderLeftWidth: Sizing._8,
-    padding: Sizing._20,
-    paddingLeft: Sizing._12, // Adds with borderLeftWidth for 20
+    borderLeftWidth: spacing.vadsSpaceXs,
```

Comment:
Spacing tokens should not be used for border widths. In the future, we may create tokens specifically for borderWidth.

Reply:
This is a special case because how react native handles things. If you note the other properties here, the overall padding is 20, but the left padding is 12 because it's additive with the border width to achieve 20 padding on the left side so for practical purposes this is effectively padding. It's just special padding to have the different color sidebar on the Alert.

I think it makes sense until there's a dedicated token. Even then, this is being used as padding more than a border since it's only on one side. 

Unless Narin thought otherwise, inclined to leave it as spacing.

Reply:
I ran into the same thing with the red left border for the error state in Checkbox. As Tim mentioned, in figma, borders are inset and its width is included in the padding calculation whereas in RN, they are separate.

I don't feel super strongly but I think I'm also leaning towards leaving this until we have dedicated tokens as it does serve as padding in a way. Also best to use tokens instead of absolute numbers when possible.